### PR TITLE
SW-2482 Fix ghost area where page card was not clickable

### PR DESCRIPTION
**Pull request by karthikbtf**
- moved margin to the wrapper around link
- otherwise the margin was within wrapper which does not allow clicks to escape
- also update npm version for web-components

### CODE REVIEW

**Comment by karthikbtf** [comment]
@constanzauanini I can bump this to version 30 after you land your web-components PR

**Comment by constanzauanini** [comment]
> @constanzauanini I can bump this to version 30 after you land your web-components PR

Merged! Thanks

**Review by constanzauanini — APPROVED** [review]

**Comment by karthikbtf** [comment]
> > @constanzauanini I can bump this to version 30 after you land your web-components PR
> 
> Merged! Thanks

👍 will wait for new npm release and update this PR

**Pull request merged**
